### ISSUE TITLE
Game freezes as soon as the world loads and I get ingame. Crashes ~7 seconds after this.

### ISSUE THREAD

**Issue by TimmehTheMeme** [AllTheMods/ATM-3#812]
If you need more info aside from the [Crash Report](https://hastebin.com/vefiyowali.hs) just ask.

**Comment by Yoosk** [comment]
You should update your java to latest, allocate more ram, update graphic driver.

As for error, not sure what is the cause, but it might be due to capped ram and/or some driver issue.

**Comment by FallenEagle1** [comment]
Looking at the forge version (2705) it also looks like you are running an older version of the modpack, so I would suggest updating the modpack as well.

**Comment by TimmehTheMeme** [comment]
> Looking at the forge version (2705) it also looks like you are running an older version of the modpack, so I would suggest updating the modpack as well.

Updating it results in a corrupted world and awful FPS.

It's been working fine for months.

I will try allocating more RAM though.

**Comment by FallenEagle1** [comment]
Updating should not result in a corrupted world. Where are you downloading the modpack from, and what version are you starting with, and then going to?

**Comment by TimmehTheMeme** [comment]
From the twitch launcher. 

I tried updating and when I load into said world there are some pretty serious issues. 

1. Purple/Black texture errors.
2. All my pipes, cables etc have been disconnected and/or replaced with texture errors.
3. My FPS tanks and it is unplayable.

Going from v. 5.9.6. I tried every  version up to 5.11.2 to no success. Two or 3 versions up is fine but I still get the occasional texture error.

I have tried a clean install but that results in me not even being able to load the world. It simply black screens and returns to the menu.

**Comment by Yamza** [comment]
The only thing called out in your previous log was a thaumcraft particle.
`thaumcraft.client.fx.ParticleEngine.updateParticles`

**Comment by FallenEagle1** [comment]
Wait.... Looked a little closer, and that is a thaumcraft crash. Besides updating, there is not much that can be done about it. That crash was fixed in a newer version of the mod (which is in the updated modpack) So you'll have to figure something out to that effect. If you are going to update from 5.9.6, maybe start with 5.9.7ish or 5.9.8d then update to 5.11.2. Or, just start a new world. There have been a large number of changes between the updates.
Looking at the crash report, you have 5.5 Gb allocated, which for the newer version of the modpack isn't quite enough (more mods added) so not surprised that your fps tanks. Going through this link and making the suggested changes (all of them) should help. https://www.reddit.com/r/allthemods/comments/968r4k/how_to_fix_modpack_wont_load_fps/

**Comment by TimmehTheMeme** [comment]
I have over 6GB allocated in the settings, that doesn't make sense. 

Why is this crash only happening now ? 

It was working fine yesterday, for the many months since I started this world.

**Comment by FallenEagle1** [comment]
I don't know what starts the crash, I just know it is one that exists and was fixed in updates.

**Comment by TimmehTheMeme** [comment]
Ok, any idea why updating the pack causes errors in the world ?

**Comment by Yamza** [comment]
How are you updating the pack? Did you change/add/remove any mods in the pack version you are on?

Try making a new instance of ATM3 and copying the world folder over to it.

**Comment by TimmehTheMeme** [comment]
> How are you updating the pack? Did you change/add/remove any mods in the pack version you are on?
> 
> Try making a new instance of ATM3 and copying the world folder over to it.

Already tried a clean install, nothing changes.

On a seperate note, why does my taskmgr say that the game is using 9+ GB or RAM but my allocated is reading as only ~5.5 in the log even though I have 6 allocated in the JVM arguments.

**Comment by FallenEagle1** [comment]
there are some background java things that tend to run also. Add small memory leak and you get that. The link I posted before will help with that, making sure it gets cleaned up more. It wont fix it completely though. It just might be that that was an old enough world that its incompatible. Updating more often would help with this.

**Comment by TimmehTheMeme** [comment]
OK I HAVE FOUND THE ISSUE.

This is pretty weird. I'll try and explain. 

When I last logged on to my world, the game crashed as I was doing something stupid with a mekanism multiblock. This was in the OVERWORLD. 

I managed to get into the game by a fresh install and a lucky load on a different version. 

Turns out, the reason I was crashing was because the game decided to spawn me in an unpowered rftools dimension. This is odd because that dimension is my lithium tower dimension and is ALWAYS powered. I died, I re-powered the dimension and reclaimed my stuff. 

It's all good, thank you all for your help. It was just modded being modded in the end, 1 in a million bug.
